### PR TITLE
Hotfix/marketing hero

### PR DESCRIPTION
**Pull request by tomrndom**
**What kind of change does this PR introduce?**

* Fix checking image quantity for use of slider
* Adjusting slider height dynamically

**Does this PR introduce a breaking change?**

None

**What needs to be documented once your changes are merged?**

None

### CODE REVIEW

File: src/components/MarketingHeroComponent.js [hunk 1]
```diff
@@ -25,117 +40,113 @@ class MarketingHeroComponent extends React.Component {
...
-      </section>
-    );
-  }
+        <div className={`${styles.rightColumn} column is-6 px-0`} id="marketing-slider" ref={sliderRef}>
+          {siteSettings.heroBanner.images.length > 1 ?
+            <Slider {...sliderSettings}>
+              {siteSettings.heroBanner.images.map((img, index) => {
+                return (
+                  <div key={index}>
+                    <div className={styles.imageSlider} style={{ backgroundImage: `url(${img.image})`, height: sliderHeight, marginBottom: -6 }} />
```
Comment (smarcet) [inline src/components/MarketingHeroComponent.js:136]: originally sliderHeight is null is that correct ?

Comment (tomrndom) [reply src/components/MarketingHeroComponent.js:136]: I'll set a default value using the old hardcoded value

**Review by smarcet — CHANGES_REQUESTED** [review]
@tomrndom please review comments

**Review by smarcet — APPROVED** [review]
looks good to me

**Pull request merged**